### PR TITLE
gas tests and don't use optimizations

### PR DESCRIPTION
**Pull request by kmbarry1**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,7 @@
-build  :; DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1000000 dapp --use solc:0.6.12 build
-clean  :; dapp clean
-test   :; DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1000000 dapp --use solc:0.6.12 test -v ${TEST_FLAGS}
+.PHONY: build clean test test-gas
```
Comment (gbalabasquer) [inline Makefile:1]: Just learnt what this what about.

**Comment by gbalabasquer** [comment]
Please comment this line as well: https://github.com/makerdao/dss/blob/liq-2.0-gas/shell.nix#L8

**Comment by kmbarry1** [comment]
> Please comment this line as well: https://github.com/makerdao/dss/blob/liq-2.0-gas/shell.nix#L8

Eek, that needs to use the latest flags as well, `SOLC_FLAGS` is deprecated since `dapp` version `0.32.0`.

**Comment by kmbarry1** [comment]
> > Please comment this line as well: https://github.com/makerdao/dss/blob/liq-2.0-gas/shell.nix#L8
> 
> Eek, that needs to use the latest flags as well, `SOLC_FLAGS` is deprecated since `dapp` version `0.32.0`.

Although...if it's using `hevm` `0.43.1`...I think it's still on a pre-`0.32.0` `dapp` version? `hevm` `0.44.0` and `0.32.0` came together.

**Comment by kmbarry1** [comment]
> > > Please comment this line as well: https://github.com/makerdao/dss/blob/liq-2.0-gas/shell.nix#L8
> > 
> > 
> > Eek, that needs to use the latest flags as well, `SOLC_FLAGS` is deprecated since `dapp` version `0.32.0`.
> 
> Although...if it's using `hevm` `0.43.1`...I think it's still on a pre-`0.32.0` `dapp` version? `hevm` `0.44.0` and `0.32.0` came together.

Okay, yes, confirmed this suspicion...we should consider separately whether to upgrade the version of `dapp` used in `shell.nix`.

File: Makefile (hunk 1 above)
Comment (kmbarry1) [reply Makefile:1]: I only recently looked it up myself :sweat_smile:

**Comment by gbalabasquer** [comment]
Right, it's an older dapptools version. Yeah at some point we should upgrade it.

**Review by gbalabasquer — APPROVED** [review]
LGTM

**Review by e18r — APPROVED** [review]
I agree, optimizations aren't worthwhile.

**Review by godsflaw — APPROVED** [review]
LGTM

**Pull request merged**